### DISCUSSION TITLE
Docker-compose haugene/transmission-openvpn-proxy / Privoxy?

### DISCUSSION THREAD

**Discussion by thijskaspers** [haugene/docker-transmission-openvpn#1971] [Q&A] [answered]
Hey there,

I was wondering if the `haugene/transmission-openvpn-proxy` is still needed? That is still in the docker-compose file in the repository for a proxy container, but in the documentation there is no mention of this anymore and it has a page about Privoxy that is added to the main container. Can I use Privoxy to access Transmission from another network, or are the OpenVPN proxy and Privoxy two different things?

Could someone clarify this for me?

Thanks!

**Comment by pkishino** [comment, marked as answer]
the proxy is for reverse proxying traffic from other containers
https://haugene.github.io/docker-transmission-openvpn/vpn-networking/
the privoxy is for proxying your web-traffic
https://haugene.github.io/docker-transmission-openvpn/web-proxy/
